lbrynet/daemon/Daemon.py

```diff
@@ -2128,6 +2128,22 @@ def jsonrpc_transaction_show(self, txid):
         return d
 
     @AuthJSONRPCServer.auth_required
+    def jsonrpc_is_address(self, address):
+        """
+        Checks if the given string is a valid wallet address.
+
+        Usage:
+            is_address (<address> | --address=<address>)
+
+        Returns:
+            (bool) true, if address is valid
+        """
+
+        d = self.session.wallet.address_is_mine(address)
```

Comment:
Duh, thanks. I started by copy and pasting the code from jsonrpc_address_is_mine since it's almost the exactly the same thing, but I didn't notice that I failed to change that part because all of the test addresses that I used happened to be my own.